jikkou/kafka-topics-values.yaml

```diff
@@ -166,17 +166,25 @@ apps:
     customTopics: {}
   deduplicator:
     name: jpo-deduplicator
-    streamTopics:
-      - topic.DeduplicatedProcessedMap
-      - topic.DeduplicatedProcessedMapWKT
-      - topic.DeduplicatedOdeMapJson
-      - topic.DeduplicatedOdeTimJson
-      - topic.DeduplicatedOdeRawEncodedTIMJson
-      - topic.DeduplicatedOdeBsmJson 
-      - topic.DeduplicatedProcessedBsm
-      - topic.DeduplicatedProcessedSpat
+    streamTopics: {}
     tableTopics: {}
-    customTopics: {}
+    customTopics:
+      - topicName: topic.DeduplicatedProcessedMap
+        retentionMs: 43200000  # 12 hours
+      - topicName: topic.DeduplicatedProcessedMapWKT
+        retentionMs: 43200000
+      - topicName: topic.DeduplicatedOdeMapJson
+        retentionMs: 43200000
+      - topicName: topic.DeduplicatedOdeTimJson
+        retentionMs: 43200000
+      - topicName: topic.DeduplicatedOdeRawEncodedTIMJson
+        retentionMs: 43200000
+      - topicName: topic.DeduplicatedOdeBsmJson
+        retentionMs: 43200000
+      - topicName: topic.DeduplicatedProcessedBsm
+        retentionMs: 43200000
+      - topicName: topic.DeduplicatedProcessedSpat
+        retentionMs: 43200000
```

Comment:
This works, but just FYI this entire file is overridden in the CDOT deployment, so the config map mounted in CDOT's deployment will also need to be updated.

Reply:
Oh, cool! I didn't see that until now, good thinking!